Tests/LibGfx/TestImageWriter.cpp

```diff
@@ -113,20 +113,30 @@ TEST_CASE(test_bmp)
 TEST_CASE(test_gif)
 {
     // Let's limit the size of the image so every color can fit in a color table of 256 elements.
-    auto bitmap = TRY_OR_FAIL(TRY_OR_FAIL(create_test_rgb_bitmap())->cropped({ 0, 0, 16, 16 }));
+    auto bitmap_1 = TRY_OR_FAIL(TRY_OR_FAIL(create_test_rgb_bitmap())->cropped({ 0, 0, 16, 16 }));
+    auto bitmap_2 = TRY_OR_FAIL(TRY_OR_FAIL(create_test_rgb_bitmap())->cropped({ 16, 16, 16, 16 }));
 
-    auto encoded_bitmap = TRY_OR_FAIL((encode_bitmap<Gfx::GIFWriter>(bitmap)));
-    auto decoder = TRY_OR_FAIL(Gfx::GIFImageDecoderPlugin::create(encoded_bitmap));
+    auto stream_buffer = TRY_OR_FAIL(ByteBuffer::create_uninitialized(10 * 1024));
+    FixedMemoryStream stream { Bytes { stream_buffer } };
+    auto animation_writer = TRY_OR_FAIL(Gfx::GIFWriter::start_encoding_animation(stream, bitmap_1->size()));
```

Comment:
Maybe it's nice to keep the test for GIFImageDecoderPlugin::create()? Consider adding a `test_animated_gif` that does what the RHS does and keep the exisiting `test_gif` as-is.

Reply:
Good point, I didn't think of that